### DISCUSSION TITLE
How to make cursor move to another display (workspace) only by pressing key stroke?

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#4920] [Q&A]
Hello.

In multi-monitor configurations it's not always comfortable to control mouse cursor stay in on display. Ho to make cursor move to another workspace only by keystroke combination pressed, like [meta] + [n]?

**Comment by Airblader** [comment]
`xdotool` can do that for you (and then bind that command in your i3 config).

**Reply by ghost** [reply]
How `xdotool` can help?

**Reply by Rom1deTroyes** [reply]
xdotool can move your mouse to a defined screen, desktop, window, ...
So you can have something like :
```i3
bindsym $mod+n exec xdotool mousemove --screen 0 'restore'
```
(I have only one screen under the hand, so I cannot test this, but you have the idea :-)

**Reply by kcrmson** [reply]
I feel this would do the job, it seems to perform as you like. In my two monitors + the laptop internal display, using any of these three keybinds moves the focus to that monitor and if I move the mouse I notice it's already there due to a workspace there being focused (even if it's currently empty). Change monitors to the appropriate names, keybinds to whatever keys are comfy:

```
# monitor variables
set $mon1 "DP-3.2"
set $mon2 "eDP-1-1"
set $mon3 "DP-3.1"

# display output navigation
bindsym $mod+n exec --no-startup-id i3-msg focus output $mon1
bindsym $mod+m exec --no-startup-id i3-msg focus output $mon2
bindsym $mod+comma exec --no-startup-id i3-msg focus output $mon3

```
